vendor/github.com/go-mysql-org/go-mysql/schema/schema.go

```diff
@@ -385,14 +422,23 @@ func (ta *Table) fetchIndexesViaSqlDB(conn *sql.DB) error {
 			return errors.Trace(err)
 		}
 
-		if currentName != indexName && visible == "YES" {
```

Comment:
Comment purely for diff visibility. Previously we were returning only visible indexes, now the filtering happens later in `LoadTables` 👀